cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/clinical/CVRClinicalDataProcessor.java

```diff
@@ -54,12 +55,21 @@
     @Autowired
     public CvrSampleListUtil cvrSampleListUtil;
     
+    Logger log = Logger.getLogger(CVRClinicalDataProcessor.class);
+
     @Override
     public CompositeClinicalRecord process(CVRClinicalRecord i) throws Exception {
-        List<String> record = new ArrayList<>();
+        if (i == null) return null;
```

Comment:
L62 and L68 have what looks like one extra leading whitespace?

On L69 could you also move the `catch` statement one line above? See example below

```
try {
 .....
} catch (....) {
}
```